chrome-launcher.ts

```diff
@@ -79,7 +81,7 @@ async function launch(opts: Options = {}): Promise<LaunchedChrome> {
     return instance.kill();
   };
 
-  return {pid: instance.pid!, port: instance.port!, kill};
+  return {pid: instance.pid!, port: instance.port!, kill, chromeProcess: instance.chrome!};
```

Comment:
let's rename this guy to just `process`.

Reply:
+1 will do thanks for the review!